RevenueCatUI/CustomerCenter/Data/CustomerCenterActionPreferenceConnector.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  CustomerCenterActionPreferenceConnector.swift
+//  
+//  Created by Cesar de la Vega on 2024-06-17.
+
+import RevenueCat
+import SwiftUI
+
+#if os(iOS)
+
+/// A view modifier that connects CustomerCenterViewModel actions to the SwiftUI preference system
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
```

Comment:
```suggestion
@available(iOS 15.0, *)
```
As it's specified below that it's `unavailable` for other platforms. Could be confusing otherwise